content/en/agent/configuration/secrets-management.md

```diff
@@ -17,7 +17,7 @@ The Datadog Agent allows you to securely manage secrets by integrating with any
 
 ### How it works
 
-To reference a secret in your configuration, use the `ENC[<secret_id>]` notation. This tells the Agent to resolve the value using your configured secret retrieval executable. The secret is fetched and loaded into memory but is never written to disk or sent to the Datadog backend.
+To reference a secret in your configuration, use the `ENC[<secret_id>]` notation. This tells the Agent to resolve the value using either the embedded `datadog-secret-backend` or your configured secret retrieval executable. The backend resolves the placeholder using your secret store and injects the plain-text value into the config at runtime. The secret is fetched and loaded into memory but is never written to disk or sent to the Datadog backend.
```

Comment:
Let's not mentioned `datadog-secret-backend`, it means nothing to customers. We should instead speak of "backend supported by the Agent"/"OOTB support"/...

I'm not sure `he backend resolves the placeholder using your secret store and injects the plain-text value into the config at runtime.` adds a lot here, WDYT ?